core/src/main/java/org/opensearch/sql/expression/aggregation/Aggregator.java

```diff
@@ -121,4 +128,13 @@ public boolean conditionValue(BindingTuple tuple) {
     return ExprValueUtils.getBooleanValue(condition.valueOf(tuple));
   }
 
+  private Boolean duplicated(ExprValue value, S state) {
```

Comment:
Two concers
1. the duplicated function will make the time complicity as O(N * C), C is the cardinality.
2. why not move to count in case we only support distinct count?